docs-ref-conceptual/upcoming-breaking-changes.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Upcoming Breaking Changes - Azure CLI | Microsoft Docs
+description: Learn about breaking changes coming to Azure CLI in the next breaking change release
+manager: mkluck
```

Comment:
Please remove `manager`, `author`, `ms.author`, `ms.tool` and `keywords`.

`Author` and `ms.author` come from `docfx.json` now, and `manager`, `ms.tool` and `keywords` have been deprecated.

Reply:
Done